README.md

```diff
@@ -67,3 +67,21 @@ In the platform, the log looks like as the following JSON Object:
     }
 }
 ```
+
+## How to release the NuGet package
```

Comment:
I would just explain `How to build the package` and keep the publish logic private